claroty/info.json

```diff
@@ -0,0 +1,1169 @@
+                },
+                "site_id": "",
+                "approved": "",
+                "hostname": "",
+                "os_build": "",
+                "protocol": [
+                ],
+                "last_seen": "",
+                "site_name": "",
+                "subnet_id": "",
+                "timestamp": "",
+                "asset_type": "",
+                "class_type": "",
+                "first_seen": "",
+                "network_id": "",
+                "risk_level": "",
+                "criticality": "",
+                "resource_id": "",
+                "subnet_type": "",
+                "asset_type__": "",
+                "display_name": "",
+                "purdue_level": "",
+                "special_hint": "",
+                "criticality__": "",
+                "edge_last_run": "",
+                "project_parsed": "",
+                "special_hint__": "",
+                "default_gateway": "",
+                "os_architecture": "",
+                "os_service_pack": "",
+                "virtual_zone_id": "",
+                "domain_workgroup": "",
+                "custom_attributes": [
+                ],
+                "virtual_zone_name": "",
+                "active_tasks_names": [
+                ],
+                "custom_informations": [
+                ],
+                "installed_antivirus": "",
+                "active_queries_names": [
+                ]
+              }
+            ],
+            "count_total": "",
+            "count_in_page": "",
+            "count_filtered": ""
+          }
+        },
+        {
+          "condition": "{{format === insight_assets}}",
+          "output_schema": {
+            "objects": [
+              {
+                "id": "",
+                "os": "",
+                "mac": [
+                ],
+                "ipv4": [
+                ],
+                "name": "",
+                "ghost": "",
+                "vendor": "",
+                "site_id": "",
+                "site_name": "",
+                "asset_type": "",
+                "network_id": "",
+                "risk_level": "",
+                "criticality": "",
+                "resource_id": "",
+                "asset_type__": "",
+                "criticality__": ""
+              }
+            ],
+            "count_total": "",
+            "count_in_page": "",
+            "count_filtered": ""
+          }
+        },
+        {
+          "condition": "{{format === rids}}",
+          "output_schema": {
+            "objects": [
+              {
+                "resource_id": ""
+              }
+            ],
+            "count_total": "",
+            "count_in_page": "",
+            "count_filtered": ""
+          }
+        }
+      ],
+      "title": "Get Assets",
+      "description": "Get assets from claroty server based on input parameters you have provided.",
+      "enabled": true,
+      "parameters": [
+        {
```

Comment:
please take command filter like criticality, last seen etc as input parameters to the action and build query in operations.py. Also provide extra param for open query.